code/game/bg_misc.c

```diff
@@ -1095,17 +1095,11 @@ qboolean	BG_PlayerTouchesItem(playerState_t *ps, entityState_t *item, int atTime
 
     BG_EvaluateTrajectory(&item->pos, atTime, origin);
 
-    /* we are ignoring ducked differences here */
-    if (ps->origin[0] - origin[0] > 44
-        || ps->origin[0] - origin[0] < -50
-        || ps->origin[1] - origin[1] > 36
-        || ps->origin[1] - origin[1] < -36
-        || ps->origin[2] - origin[2] > 36
-        || ps->origin[2] - origin[2] < -36) {
-        return qfalse;
-    }
+    int xDist = abs(ps->origin[0] - origin[0]);
+    int yDist = abs(ps->origin[1] - origin[1]);
+    int zDist = abs(ps->origin[2] - origin[2]);
 
-    return qtrue;
+    return (xDist < 50 && yDist < 36 && zDist < 36);
```

Comment:
I'm thinking that was an oversight on their part. Why should the distance be different forward and back on the x axis?